packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -191,9 +191,17 @@ class GuildChannelManager extends CachedManager {
   async createWebhook(channel, name, { avatar, reason } = {}) {
     const id = this.resolveId(channel);
     if (!id) throw new TypeError('INVALID_TYPE', 'channel', 'GuildChannelResolvable');
+
     if (typeof avatar === 'string' && !avatar.startsWith('data:')) {
       avatar = await DataResolver.resolveImage(avatar);
     }
+
+    const invalidNames = ['everyone', 'here'];
+    const invalidSubstrings = ['@', '#', ':', '```', 'discord', 'clyde'];
+    if (invalidNames.includes(name) || invalidSubstrings.some(c => name.includes(c))) {
+      throw new TypeError('INVALID_WEBHOOK_NAME');
```

Comment:
Is this error message registered? If not, it should be

Reply:
It is not

Reply:
Now it is, thanks for pointing this out! :)